### PR TITLE
Release 2023-11-29

### PR DESCRIPTION
**Pull request by bskinn**
Live build for #802 and #791 & post-merge fixes, plus several Dependabot updates.

### CODE REVIEW

File: apps/labs/posts/categories.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
   "Community",
   "Developer workflows",
   "Funding",
+  "GPU",
```
Comment (gabalafou) [inline apps/labs/posts/categories.json:8]: I'm assuming that the addition of this category was previously discussed, reviewed and approved

**Review by gabalafou — APPROVED** [review]
The build seems fine. I clicked on the blog posts and the new category. Looks good

File: apps/labs/posts/categories.json (hunk 1 above)
Comment (bskinn) [reply apps/labs/posts/categories.json:8]: Given that there was an additional `CuPy` category _also_ added, and then removed after subsequent discussion, I'm assuming that yes, the `GPU` category add is good to remain.

Comment (rgommers) [reply apps/labs/posts/categories.json:8]: That one looked like a good idea to me, yes. I didn't see anyone else comment on it.

Comment (bskinn) [reply apps/labs/posts/categories.json:8]: Are there any other posts in the archive that should have `GPU` backfilled onto?

**Pull request merged**